sematic/config/config.py

```diff
@@ -109,15 +112,23 @@ def server_url(self) -> str:
 
     @property
     def api_url(self) -> str:
-        return urljoin(
-            self.server_url,
-            "api/v{}".format(self.api_version),
+        return urljoin(self.server_url, "api/v{}".format(self.api_version))
```

Comment:
Shall we use f strings instead?

Reply:
Done. Propagated everywhere in touched files.